### PR TITLE
exponential bucket distribution stats support added

### PR DESCRIPTION
**Pull request by costa**
Please tag 0.3.0 when you merge

### CODE REVIEW

**Review by rachelri — APPROVED** [review]

**Comment by rachelri** [comment]
you mean before I merge right?
I created a release (and tag), let me know if it's OK so I will merge

**Comment by costa** [comment]
perfect

**Pull request merged**
